addons/viewdistance/stringtable.xml

```diff
@@ -0,0 +1,103 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Zeus">
+        <Container ID="ZeusModule">
```

Comment:
Package and container entries are wrong. In fact most ACE module don't use the container entry, as it has no use